bundle/src/main/java/com/adobe/acs/commons/data/Variant.java

```diff
@@ -64,9 +66,13 @@ public Variant() {
     public <T> Variant(T src) {
         setValue(src);
     }
-
+    
     public Variant(Cell src) {
-        setValue(src);
+        this(src, Locale.getDefault());
```

Comment:
Wouldn't it make more sense to take the Locale.ROOT as a language/country agnostic default?

Reply:
It could be, but it would change current behavior. This is a public constructor, so clients may already be using it and relying on the current default of Locale.getDecault().